api/v1/helpers/verbs/doPost.js

```diff
@@ -97,6 +96,10 @@ function doPost(req, res, next, props) {
   }
 
   return postPromise.then((o) => {
+    if (!o) { // could be here from u.handleError
+      return;
```

Comment:
Currently refactoring the doPost function to make it more testable. There will be a lot more tests coming.